### PR TITLE
Fix and test PSK, simplify docs, remove inconsistent snake_case

### PR DESCRIPTION
**Pull request by emschwartz**

### CODE REVIEW

File: src/lib/receiver.js [hunk 1]
```diff
@@ -170,13 +176,13 @@ function createReceiver (opts) {
...
    *
-   * @return {Object} Object containing destination address and shared secret
+   * @return {PSKParams}
    */
   function generateSharedSecret () {
     const token = base64url(hmacHelper.getPskToken())
     return {
-      destination_account: getAddress() + '.' + pskReceiverId + token,
-      shared_secret: base64url(hmacHelper.getPskSharedSecret(token))
+      destinationAccount: getAddress() + '.' + pskReceiverId + token,
```
Comment (justmoon) [inline src/lib/receiver.js]: Hmm, the snake case here wasn't random, I was thinking that you might use these values in other applications, so the field names should be standardized. It's obviously not a big deal, it's easy enough to change it manually if another implementation expects `destination_account`.

We decided to use snake case for all protocols. The question is whether JSONifying these values and copying them into a config file somewhere constitutes a "protocol".

File: src/lib/receiver.js [hunk 2]
```diff
@@ -26,6 +26,12 @@ const PSK_RECEIVER_ID_PREFIX = '~psk.'
  */
 
 /**
+ * @typedef {Object} PSKParams
```
Comment (justmoon) [inline src/lib/receiver.js]: Should probably be `PskParams` - not saying the code is currently consistent, but I'm seeing more HttpRequest than HTTPRequest.

File: src/lib/receiver.js (hunk 1 above)
Comment (emschwartz) [inline src/lib/receiver.js]: For now I don't think this return value constitutes a protocol by itself so I think we should go with camelCase

File: src/lib/receiver.js (hunk 2 above)
Comment (emschwartz) [inline src/lib/receiver.js]: Fixed

File: README.md [hunk 3]
```diff
@@ -114,7 +59,7 @@ const co = require('co')
 const ILP = require('ilp')
 const FiveBellsLedgerPlugin = require('ilp-plugin-bells')
 
-const sender = ILP.createSender({
+const sender = ILP.createSender(i
```
Comment (sharafian) [inline README.md]: Typo

**Pull request merged**